docs/guides/force-organizations.mdx

```diff
@@ -117,29 +161,37 @@ export const CustomSwitcher = () => {
 };
 
 ```
-    </Tab>
 
-    <Tab>
-```tsx filename="app/[orgId]/sync-active-organization.tsx"
+#### Set an active organization based on the URL
```

Comment:
This is a great section @panteliselef and the code example is solid. However there is nothing instructing the developer on how to use this component or where it should be mounted.

I think this should be present as an alternative to CustomOrgSwitcher. Show how to mount this in 'pp/(with-active-organization)/layout.tsx' and explain its tole versus a component to let the user select an org. That, or cover its use in some way.

Reply:
I forgot to mention this as a comment last night when I posted this PR! Thank you Roy. And I am giving this a +1 that we should show where this `SyncActiveOrganization` component can be used/mounted.